models/lavender_haze/configs/config_hyperparameters.py

```diff
@@ -1,14 +1,22 @@
 def get_hp_config(): 
     hp_config = {
-        "clf":{
-            "learning_rate": 0.05,
-            "n_estimators": 100,
-            "n_jobs": 12
-        },
-        "reg":{
-            "learning_rate": 0.05,
-            "n_estimators": 100,
-            "n_jobs": 12
+        "name": "lavender_haze",
```

Comment:
We need to think here a bit. Because the config_hyperparameters is meant to only contain "operational" parameters. I.e. parameters you can actually change here to change to model. And I'm not sure that is the case for:

       "algorithm": "HurdleRegression",
        "model_clf": "LGBMClassifier",
        "model_reg": "LGBMRegressor",
        "depvar": "ln_ged_sb_dep",
        
I mean, could I change any of these just here, and then get a different but still functional model?

I think I have produced some writing somewhere about it, but given what we talked about yesterday I'll make that more clear.

Reply:
Yes, changing these parameters will produce a different model.